pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.flake8]
+# TODO add python directory here in later PR
+filename = ["./elements/**/*.py", "./question-servers/**/*.py"]
+ignore = ["E501"]
+# Black compatibility
+max-line-length = 88
+extend-ignore = ["E203", "W503"]
+
+[tool.black]
+include = '^\/elements\/.*\.py$|^\/question-servers\/.*\.py$'
```

Comment:
Note that (for some reason) black uses a pure regex to select files for the include, which differs from the way that flake8 and isort are configured.